src/common/Formatter.cc

```diff
@@ -296,7 +296,18 @@ void JSONFormatter::finish_pending_string()
   }
 }
 
-template <class T>
+void JSONFormatter::add_value(std::string_view name, double val) {
+  CachedStackStringStream css;
+  if (std::isinf(val) || std::isnan(val)) {
```

Comment:
Would the shorter `!std::isfinite(val)` form work here?

Reply:
Sorry, missed your comment somehow.
It does work but do you have any special observation for using that instead of `std::isinf(val)` ?

Reply:
It's shorter, expresses the intent more directly, enumerates goodness not badness,... i.e. purely aesthetical reasons.